src/routes/docs/advanced/security/backups/+page.markdoc

```diff
@@ -10,5 +10,10 @@ Backups are crucial for:
 - Recovering in case of malicious alteration or deletion of data.
 - Reducing downtime in the event of accidental data loss, corruption, or malicious attacks.
 
-Appwrite Cloud uses multiple databases for its products. Some data bases will have access to point-in-time recovery from the past 7 days.
-Other databases performs regular data backups every 4 hours. Each backup is kept for 7 days.
\ No newline at end of file
+Appwrite Cloud provides both automated and self-managed backups.
+Users can configure automatic backup policies or initiate manual backups directly from the Console.
```

Comment:
Let's clarify these self-managed backups are for pro plans and above and link to the backups documentation here.